Formula/widelands.rb

```diff
@@ -20,7 +19,9 @@ class Widelands < Formula
     sha256 x86_64_linux:   "1032792700cccdbaa04827dbc6849c309fa75d649f3253bd0c7568a1fd5d6770"
   end
 
+  depends_on "asio" => :build
   depends_on "cmake" => :build
+
   depends_on "boost"
```

Comment:
Do we still need Boost?

Reply:
per [this issue](https://github.com/widelands/widelands/issues/5115), no, let me update the list and make some run